docs/operations-guide/running-metabase-on-azure.md

```diff
@@ -0,0 +1,183 @@
+1) database private IP address: 10.0.2.4
+2) database port: 5432 (in the case of Postgres, MySQL/MariaDB default port is 3306)
+3) name of the database server: metabase-app-database
+4) username of the database: metabase
+5) password: Password1!
+
+then my connection string will be: `postgresql://10.0.2.4:5432/postgres?user=metabase@metabase-app-database&password=Password1!&ssl=true&sslmode=require`
+
+Click on save and the instance will restart. After this step you will be able to access your Metabase in the URL that you have in the "Overview" tab in the web app under the URL section (will be the name you chose in the first step of the config of the web app + .azurewebsites.net as Azure handles all the deployment and domain)
+
+## Additional configurations
+### How to enable Health checks
+
+Enabling health checking in Metabase is a good practice. You can tell the "host" when Metabase is ready to server your customers by just doing a simple configuration: go to your web app -> Monitoring -> Health Check -> Enable health check and include in the path `/api/health`.
+
+### How to update
+
+Simply go to your web app and in Settings -> Container Settings (Classic) and there change the image version of Metabase.
+In the same way you do an upgrade, you can change the Metabase version to Enterprise by switching to v1 instead of v0 in the prefix of the docker image.
+
+**Important**: always ensure you have a backup of your Database before doing a version upgrade, *specially*, when upgrading between major versions.
+
+### How to see the logs
+
+If you want to see the logs of your deployment, you need to go to the web app that you created and you will be able to see the streaming logs in Monitoring -> Log stream.
+
+### Performance tuning
+
+- Disable FTP State
+- Change HTTP version to 2.0
+- Enable WebSockets
+- ARR Affinity enabled in case you scale out
+
+### Custom domains
+
+You can go to Settings -> custom domains in your web app use a custom domain that you have in Azure, with your own certificates
+
+### Scale-up vs Scale-out
+
+Azure provides an easy way in which you can increase the server's horsepower by doing a Scale-up. Simply choose a pricing tier with more RAM and CPU count and restart the instance. This approach is easier than scaling out and you should choose this option first before scaling out.
+
+In case you know what you are doing and want to have a highly-available Metabase deployment, you can tell your web app to scale-out (more servers serving the application in parallel). Simply go to Scale-out and use the slider to ask for more instances or use a custom autoscaling policy (like more servers during your work hours and less during night, or anything you come up with)
+
+### CORS
+
+In case you are using embedding, you might have to enable CORS in Settings -> CORS in the web application. You will need to fill the origin domains from where you are embedding so Azure knows that they need to enable cross requests from there.
```

Comment:
```suggestion
In case you're embedding Metabase, you might need to enable CORS in **Settings** -> **CORS** in the web application. You'll need to fill the origin domains for the app with the embedded Metabase, so Azure knows to enable cross requests from those domains.
```